src/Constants/Options.ts

```diff
@@ -0,0 +1,226 @@
+  { value: 'UK-11-Photon-med-23', label: 'UK-11-Photon-med-23' },
+  { value: 'UK-11-Catalyst-med-23', label: 'UK-11-Catalyst-med-23' },
+  { value: 'UK-11-Alpha-eng-23', label: 'UK-11-Alpha-eng-23' },
+  { value: 'UK-11-Photon-eng-23', label: 'UK-11-Photon-eng-23' },
+  { value: 'UK-11-Catalyst-eng-23', label: 'UK-11-Catalyst-eng-23' },
+  { value: 'HR-9-Foundation-mat-22', label: '' },
+  { value: 'DL-10-Foundation-sci-22', label: '' },
+  { value: 'NEEV L2', label: 'NEEV L2' },
+  { value: 'NEEV L3', label: 'NEEV L3' },
+  { value: 'NEEV L1', label: 'NEEV L1' },
+  { value: 'EN-10-Registration-22', label: 'EN-10-Registration-22' },
+  { value: 'EN-10-Registration-23', label: 'EN-10-Registration-23' },
+  { value: 'AF-100-physics-22', label: 'AF-100-physics-22' },
+  { value: 'AF-100-chemistry-22', label: 'AF-100-chemistry-22' },
+  { value: 'AF-100-maths-22', label: 'AF-100-maths-22' },
+  { value: 'AF-100-biology-22', label: 'AF-100-biology-22' },
+  { value: 'AF-100-physics-23', label: 'AF-100-physics-23' },
+  { value: 'AF-100-chemistry-23', label: 'AF-100-chemistry-23' },
+  { value: 'AF-100-maths-23', label: 'AF-100-maths-23' },
+  { value: 'AF-100-biology-23', label: 'AF-100-biology-23' },
+  {
+    value: 'CA-100-Hiring-physics-PCMB-23',
+    label: 'CA-100-Hiring-physics-PCMB-23',
+  },
+  { value: 'CA-100-Hiring-chemistry-23', label: 'CA-100-Hiring-chemistry-23' },
+  { value: 'CA-100-Hiring-maths-23', label: 'CA-100-Hiring-maths-23' },
+  { value: 'CA-100-Hiring-biology-23', label: 'CA-100-Hiring-biology-2' },
+  { value: 'CA-100-Hiring-botany-23', label: 'CA-100-Hiring-botany-23' },
+  { value: 'CA-100-Hiring-zoology-23', label: 'CA-100-Hiring-zoology-23' },
+  { value: 'HP-10-Foundation-sci-22', label: 'HP-10-Foundation-sci-22' },
+  { value: 'UK-10-Catalyst-Eng-23', label: 'UK-10-Catalyst-Eng-23' },
+  { value: 'AF-10-Foundation-sci-22', label: 'AF-10-Foundation-sci-22' },
+  { value: 'NEEV SCERT', label: 'NEEV SCERT' },
+];
+
+export const CourseOptions: Option[] = [
+  { value: 'NEET', label: 'NEET' },
+  { value: 'Catalyst', label: 'Catalyst' },
+  { value: 'Alpha', label: 'Alpha' },
+  { value: 'Hiring', label: 'Hiring' },
+  { value: 'Certification', label: 'Certification' },
+  { value: 'Foundation', label: 'Foundation' },
+  { value: 'Photon', label: 'Photon' },
+];
+
+export const StreamOptions: Option[] = [
+  { value: 'Medical', label: 'Medical' },
+  { value: 'Maths', label: 'Maths' },
+  { value: 'Science', label: 'Science' },
+  { value: 'Physics', label: 'Physics' },
+  { value: 'Chemistry', label: 'Chemistry' },
+  { value: 'PCMB', label: 'PCMB' },
+  { value: 'Botany', label: 'Botany' },
+  { value: 'PCMBA', label: 'PCMBA' },
+  { value: 'Zoology', label: 'Zoology' },
+];
+
+export const GradeOptions = Grades.map((g) => ({ value: g, label: g }));
+
+export const AuthOptions: Option[] = Object.values(AuthType).map((item) => ({
+  value: item,
+  label: item,
+}));
+
+export const ActiveDaysOptions: Option[] = [
```

Comment:
i feel we can provide few presets here later on. 

all selected ... MWF selected. .. etc